openstack/cross-layers/packetfunctions.c

```diff
@@ -79,6 +79,41 @@ void packetfunctions_mac16bToMac64b(open_addr_t* mac16b, open_addr_t* mac64btoWr
 
 //======= address recognition
 
+bool packetfunctions_isBroadcastMulticast_debug(open_addr_t* address, uint8_t code) {

```

Comment:
I used it for debug, to be removed
(I had several unknown address type errors, because the packet was malformed. To trace their usage, I modified the function so that I can identify quickly the faulty part)